js/ui/autocomplete.js

```diff
@@ -161,8 +161,7 @@ var Autocomplete = DropDownList.inherit({
 
     _listConfig: function() {
         return extend(this.callBase(), {
-            pageLoadMode: "none",
-            indicateLoading: false
+            pageLoadMode: "none"
```

Comment:
Why are we going to remove this?

Reply:
The value of this option is already set to "false" (in the parent component)